client/src/entrypoints/admin/preview-panel.js

```diff
@@ -177,12 +177,18 @@ function initPreview() {
           !data.is_available,
         );
 
+        if (!data.is_available) {
+          setPreviewWidth();
+        }
+
         if (data.is_valid) {
           reloadIframe();
         } else if (!cleared) {
           clearPreviewData();
           cleared = true;
           reloadIframe();
+        } else {
+          finishUpdate();
```

Comment:
```suggestion
          // Finish the process when initial data is invalid to avoid elements such as the loading spinner being shown indefinitely
          finishUpdate();
```